docs/versions/unversioned/sdk/barometer.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Barometer
+---
+
+Access the device barometer sensor to respond to changes in air pressure. `pressure` is converted to _`hectopascals`_ or _`hPa`_.
+
+| OS      | Units   | Provider                                                                                                | Description                                                                                                                         |
+| ------- | ------- | ------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| iOS     | _`hPa`_ | [`CMAltimeter`](https://developer.apple.com/documentation/coremotion/cmaltimeter)                       | Altitude events reflect the change in the current altitude, not the absolute altitude.                                              |
+| Android | _`hPa`_ | [`Sensor.TYPE_PRESSURE`](https://developer.android.com/reference/android/hardware/Sensor#TYPE_PRESSURE) | Monitoring air pressure changes.                                                                                                    |
+| Web     | `N/A`   | `N/A`                                                                                                   | This sensor is not available on the web and cannot be accessed. An `UnavailabilityError` will be thrown if you attempt to get data. |
+|         |
+
+## Usage
+
+> This module was introduced in Expo v33
+
+Barometer can be used outside of the Expo client with the `expo-sensors` & `expo-sensors-interface` libraries.
+
+## Methods
+
+### `Barometer.isAvailableAsync()`
+
+Returns a promise which resolves into a boolean denoting the availability of the device barometer.
```

Comment:
```suggestion
Returns a promise which resolves into a boolean denoting the availability of the device barometer. This does not account for whether the user has granted permission to get the barometer's measurements.
```

Reply:
I still cannot find anything regarding permission. It doesn't look like you need anything in the `info.plist`. I suppose that would mean this function accounts for all availability cases.